src/DomainManager.Bussines/Notifications/CommandConsumers/PingCommandConsumer.cs

```diff
@@ -0,0 +1,43 @@
+using DomainManager.Abstract;
+using DomainManager.Notifications.CommandConsumers.Base;
+using MassTransit.Mediator;
+using Microsoft.Extensions.Caching.Memory;
+using Telegram.Bot;
+using System.Net.NetworkInformation;
+
+namespace DomainManager.Notifications.CommandConsumers;
+
+public class PingCommandConsumer : CommandConsumerBase, IMediatorConsumer {
+    private readonly ApplicationDbContext _db;
+    private readonly IScopedMediator _mediator;
+
+    public PingCommandConsumer(ITelegramBotClient botClient, ApplicationDbContext db, IScopedMediator mediator,
+        IMemoryCache memoryCache) :
+        base(Command.PingCommand, botClient, memoryCache) {
+        _db = db;
+        _mediator = mediator;
+    }
```

Comment:
зависимости не используются

```suggestion
    public PingCommandConsumer(ITelegramBotClient botClient, IMemoryCache memoryCache) :
        base(Command.PingCommand, botClient, memoryCache) { }
```